crates/core/src/address.rs

```diff
@@ -143,6 +144,9 @@ impl From<raw::Address<'_, raw::Validated>> for Address {
             raw::Discriminant::TempStorage => {
                 Address::Internal(InternalAddress::TempStorage)
             }
+            Discriminant::ReplayProtection => {
+                Address::Internal(InternalAddress::ReplayProtection)
+            }
```

Comment:
```suggestion
            raw::Discriminant::ReplayProtection => {
                Address::Internal(InternalAddress::ReplayProtection)
            }
```
yes, this is uber OCD